sbol/test/test_componentdefinition.py

```diff
@@ -164,3 +164,16 @@ def testHasUpstreamComponent(self):
         uri = 'http://sbols.org/CRISPR_Example/gRNA_b_gene/CRa_U6/1.0.0'
         c = cd.components.get(uri)
         self.assertTrue(cd.hasUpstreamComponent(c))
+
+    def testOwnedLocation(self):
```

Comment:
`createCut()` should probably be included as well, just so we don't miss it if something changes in the future.

Reply:
I had that in there originally, but then stumbled into #130.  I will add it back as a separate test and mark as expected failure.